### ISSUE TITLE
[A11y] 1.1.1 Non-Text Content – Map Name Not Meaningful

### ISSUE THREAD

**Issue by bsaadat** [PublicHealthEngland/coronavirus-dashboard#80]
When navigating the site with screen readers, the map is announced as 'Map'. As this item is required to be part of the focus order, due to having controls for interaction, the accessible name should be clear on the map's functionality - even if the user cannot see it.

It is our recommendation that the accessible name is update to be meaningful, such as 'Map showing a heat map of corona virus cases in the uk'

**This does not comply with WCAG Level A 2.1:
1.1.1 Non-Text Content** - https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html   
 
**Steps to Recreate:**
1.    Navigate to URL with screen reader enabled
2.    Listen to page overview announced by screen reader
3.    Observe issue
 
**This has been observed on the following pages:**
Coronavirus (COVID-19) cases in the UK
 
**Environment:**
Windows 10 – Internet Explorer 11, JAWS 2020
Windows 10 - Firefox, NVDA
macOS 10.14  - VoiceOver
 
**This issue is known to occur on:**
Believed to be global
 
**Reproduction rate:**
100% - Consistently reproducible. Occurs every time that it is attempted.
 
**Supporting material:**
Screenshot/Video
 
**Version:**
https://coronavirus.data.gov.uk/
 
3.Minor Impact on Functionality

**Comment by bsaadat** [comment]
![image](https://user-images.githubusercontent.com/27902577/80504343-e503ed80-896a-11ea-920a-0e309e9bfa82.png)

**Comment by bsaadat** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [[A11y] 1.1.1 Non-Text Content – Map Name Not Meaningful](https://trello.com/c/wwrQGzL9/220-a11y-111-non-text-content-map-name-not-meaningful)

**Comment by xenatisch** [comment]
Fixed in #142, with thanks to @richardTowers for the contribution.
